autometa/validation/datasets.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+COPYRIGHT
+Copyright 2021 Ian J. Miller, Evan R. Rees, Kyle Wolf, Siddharth Uppal,
+Shaurya Chanana, Izaak Miller, Jason C. Kwan
+
+This file is part of Autometa.
+
+Autometa is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Autometa is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with Autometa. If not, see <http://www.gnu.org/licenses/>.
+COPYRIGHT
+
+pulling data from google drive dataset with simulated or synthetic communities
+"""
+
+
+import gdown
+import os
+import sys
+import logging
+import pandas as pd
+
+from autometa.common.utilities import internet_is_connected
+
+logger = logging.getLogger(__name__)
+
+
+def download(type: str, size: list, file_names: list, dir_path: str) -> None:
```

Comment:
Preference thing about style but keeping consistent variable names will help others understand your code and also you when you come back to it later. 
e.g. instead of shortening 'community_type' to 'type' within "download()" just keep it "community_type". I tend to err on the side of my variable names being too descriptive
     download(type=community_type, size=community_size, file_names=file_names, dir_path=dir_path)